frontend/src/utility.ts

```diff
@@ -3,9 +3,14 @@ import _ from "lodash";
 import moment from "moment";
 import { ICount } from "@/types/DataTable";
 
-export const trackEditingUploadsBucket = `te.uploads.${
-  process.env.VUE_APP_PROJECT_DOMAIN
-}`;
+export const getProjectDomain = () => {
+  return window.location.host
+    .split(".")
+    .slice(1)
+    .join(".");
+};
+
+export const trackEditingUploadsBucket = `te.uploads.${getProjectDomain()}`;
```

Comment:
I propose to eliminate this const and compile the TE uploads bucket name in the appropriate component. It is used only once.